sovrin_client/test/cli/test_pool_upgrade_validation.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from sovrin_node.test.upgrade.conftest import validUpgrade
+from sovrin_client.test.cli.constants import INVALID_SYNTAX
+from plenum.common.util import randomString
+from sovrin_common.constants import JUSTIFICATION
+
+
+def testPoolUpgradeHasInvalidSyntaxIfJustificationIsEmpty(be, do, validUpgrade, trusteeCli):
+    validUpgrade[JUSTIFICATION] = ''
+
+    be(trusteeCli)
+    do('send POOL_UPGRADE name={name} version={version} sha256={sha256} '
+       'action={action} schedule={schedule} timeout={timeout} justification={justification}',
+       mapper=validUpgrade, expect=INVALID_SYNTAX, within=10)
+
+
+def testPoolUpgradeHasInvalidSyntaxIfJustificationIsVeryLong(be, do, validUpgrade, trusteeCli):
+    validUpgrade[JUSTIFICATION] = randomString(1001)
```

Comment:
Don't hardcode the number 1001, set a constant for max justification size and use a value greater than that